ambry-clustermap/src/main/java/com/github/ambry/clustermap/HelixClusterManager.java

```diff
@@ -1426,17 +1427,19 @@ private List<ReplicaId> initializeDisksAndReplicasOnNode(AmbryDataNode datanode,
           ensurePartitionAbsenceOnNodeAndValidateCapacity(mappedPartition, datanode,
               replicaConfig.getReplicaCapacityInBytes());
           // Create replica associated with this node and this partition
-          boolean isSealed = sealedReplicas.contains(partitionName);
+          ReplicaSealStatus replicaSealStatus = resolveReplicaSealStatus(partitionName, sealedReplicas);
           if (clusterMapConfig.clusterMapEnablePartitionOverride && partitionOverrideInfoMap.containsKey(
               partitionName)) {
             // override sealed state if PartitionOverride is enabled.
-            isSealed = partitionOverrideInfoMap.get(partitionName)
+            boolean isSealed = partitionOverrideInfoMap.get(partitionName)
                 .get(ClusterMapUtils.PARTITION_STATE)
                 .equals(ClusterMapUtils.READ_ONLY_STR);
+            // Currently only sealed state is supported by PartitionOverride.
```

Comment:
Thanks for the comment.
Is that better that we add the isPartialSealed code support here?
Even it's not supported yet and it always isPartialSealed already return false, we won't forget to change it when we starts to support partially sealed.

Reply:
fixed.